packages/foundry/contracts/README_WITCH.md

```diff
@@ -0,0 +1,99 @@
+# Auditing the Witch v2
+This contest is for the first major refactor on liquidations engine for the Yield Protocol. The existing engine has been heavily used since its launch in October 2021 and in that time we have found that liquidations have not been as fair as we would have liked. More often than not, liquidated users have lost all their collateral as we have failed to make liquidations competitive.
+
+This second version of the Witch aims to:
+1. Give us more flexibility in exploring different liquidation models, including moving away from Dutch auctions to fixed discount if there is not enough competition from liquidators.
```

Comment:
I'd rephrase to "including starting auctions at a fixed discount so most of the times vaults can be liquidated in a single transaction" or something like that.
My point is, we're not really moving away of the auctions.

Reply:
Yes, but that's orthogonal to this, maybe tomorrow we can have a call ?

Reply:
It's not orthogonal, part of the purpose of this refactor was to enable us to experiment with Compound-style liquidations. [Check this PR](https://github.com/yieldprotocol/vault-v2/pull/466).

Reply:
It is, maybe we're talking about the same thing (or not), that's why I think it's easier to discuss it on zoom :)